### ISSUE TITLE
Rendering bug, Bugged blocks. Minecraft 1.17 Optifine Pre-16

### ISSUE THREAD

**Issue by Scorpion34** [sp614x/optifine#5698]
![2021-06-08_13 00 38](https://user-images.githubusercontent.com/15971924/121234920-dcfaca80-c859-11eb-8a98-6c21d59b33ce.png)

the blocks were not placed by me they appeared in the sky randomly I'm assuming it's a rendering issue.

**Comment by KaiAF** [comment]
Send an F3 screenshot

**Comment by Scorpion34** [comment]
> Send an F3 screenshot

gotcha 
![2021-06-08_13 12 48](https://user-images.githubusercontent.com/15971924/121236146-3e6f6900-c85b-11eb-9367-e6e9a1f11f01.png)

**Comment by KaiAF** [comment]
Turn off render regions and fast render. See if that helps

**Comment by Scorpion34** [comment]
> Turn off render regions and fast render. See if that helps

kk render regions fixed it I'll try now with fast render, okay yeah render regions is the issue.

**Comment by Scorpion34** [comment]
> Turn off render regions and fast render. See if that helps

that also fixed the slime blocks, honey blocks, ice, stained glass and tinted glass

**Comment by KaiAF** [comment]
Ok. The bug is caused by render regions.

**Comment by Scorpion34** [comment]
> Ok. The bug is caused by render regions.

Yep sweet ty for quick response

**Comment by KaiAF** [comment]
This bug is also affecting String

**Comment by Scorpion34** [comment]
> This bug is also affecting String and Nether Portals
> ![image](https://user-images.githubusercontent.com/60985521/121237625-e0dc1c00-c85c-11eb-9696-86d17256beed.png)

yeah it's weird never really seen this happen before maybe the random blocks floating but with everything mainly transparent it's having issues

**Comment by Cart3r1234** [comment]
This also affects block.properties; any block you give the translucent layer to in blocks.properties has this issue, since it appears to only affect blocks with translucency, such as glass, stained glass, ice, slime blocks, honey blocks, nether portals, etc.  For some reason water remains unaffected though
